templates/filtered_resources.html

```diff
@@ -7,11 +8,19 @@
         <span class="tag is-danger">{{res.level}}</span>
         <span class="tag is-danger">{{res.type}}</span>
         <footer class="card-footer">
-                <a href={{res.url}} class="card-footer-item" rel="noreferrer noopener" target="_blank">Open Link</a>
+            <a href={{res.url}} class="card-footer-item" rel="noreferrer noopener" target="_blank">Open Link</a>
             <a target="_blank"
                 href="http://twitter.com/share?text=Hey, checkout this resource {{res.title}} via saadhan.devsindia.in&url={{res.url}}&hashtags=devsindia"
                 class="card-footer-item">Tweet</a>
         </footer>
     </div>
 </div>
-{% endfor %}
\ No newline at end of file
+{% endfor %}
+{% else %}
+<div class="icon-text">
+    <span class="icon has-text-danger">
+        <i class="fas fa-ban"></i>
```

Comment:
Instead of icons, can you go through this list of tools to find a graphics which depicts the empty state? (and is free to use)
https://gist.github.com/Bhupesh-V/8a77b85ca3103fbf157974ea6822a977

Reply:
Centralize all the 0 state content, instead of left aligned

Reply:
what do you think about these [illustrations](https://storyset.com/illustration/no-data/rafiki) for No Data ?

Reply:
The last one, "cuate" looks perfect https://storyset.com/illustration/no-data/cuate